os/dbuild.sh

```diff
@@ -26,7 +26,7 @@ BINDIR="${BUILDDIR}/output/bin"
 CONFIGDIR="${BUILDDIR}/configs"
```

Comment:
Please change "chosing" to "choosing" in the commit message.

Reply:
Done